lib/tasks/alerts/create_holiday_alerts.rake

```diff
@@ -0,0 +1,15 @@
+namespace :alerts do
+  desc 'Run alerts job'
+  task create_holiday: [:environment] do
+    puts Time.zone.now
+    schools = School.active
+    schools.each do |school|
+      puts "Running alerts for #{school.name}"
+
```

Comment:
Use school.calendar.holiday_approaching - this needs to match analytics code for now

Reply:
More than a single day too, keep trying and the alert subscription manager can insure we don't do duplicates